ui/src/app/pages/user-disabled.tsx

```diff
@@ -1,31 +1,27 @@
-import {Component} from '@angular/core';
 import * as React from 'react';
 
 import {StyledAnchorTag} from 'app/components/buttons';
 import {BoldHeader} from 'app/components/headers';
 import {PublicLayout} from 'app/components/public-layout';
 import colors from 'app/styles/colors';
-import {
-  ReactWrapperBase
-} from 'app/utils';
+import {buildPageTitleForEnvironment} from 'app/utils/title';
 
 const supportUrl = 'support@researchallofus.org';
 
-export const UserDisabled: React.FunctionComponent<{}> = () => {
-  return <PublicLayout>
-    <BoldHeader>Your account has been disabled</BoldHeader>
-    <section style={{color: colors.primary, fontSize: '18px', marginTop: '.5rem'}}>
-      Contact <StyledAnchorTag href={'mailto:' + supportUrl}>{supportUrl}</StyledAnchorTag> for
-      more information.
-    </section>
-  </PublicLayout>;
-};
+export class UserDisabled extends React.Component {
+  // TODO: esteemed reviewer, please yell at me if I don't write a paper about a global wrapper that
```

Comment:
the basic idea is that we could do like a thin wrapper around React.Component kind of like:
```typescript
export class ComponentWithPageTitle extends React.Component<{pageTitle, ...props}> {
    constructor(pageTitle, ...props) {
        super(props);
        document.title = buildPageTitleForEnvironment(pageTitle);
    }
}

export class UserDisabled extends ComponentWithPageTitle<{pageTitle}> {
    constructor(pageTitle) {
        super(pageTitle);
    }

    render() {
        [...]
    }
}
```